contrib/mungegithub/blunderbuss.yml

```diff
@@ -0,0 +1,60 @@
+prefixMap:
+  test:
+    - ixdy
+
+  pkg/admission:
+    - lavalamp
+    - smarterclayton
+
+  pkg/api:
+    - bgrant0607
+
+  pkg/apiserver:
+    - lavalamp
+    - smarterclayton
+
+  pkg/authentication:
+    - liggitt
+    - erictune
+
+  pkg/client:
+    - lavalamp
```

Comment:
I was just responding to your use of "Cc" in "Cc me and Deads2k" so you were clear that this isn't actually a Cc. I wasn't suggesting to not add the people you suggested. Sorry for being unclear.